.gitmodules

```diff
@@ -79,6 +79,9 @@
 [submodule "libraries/helpers/gamblor21_ahrs"]
 	path = libraries/helpers/gamblor21_ahrs
 	url = https://github.com/gamblor21/Gamblor21_CircuitPython_AHRS.git
+[submodule "libraries/drivers/dynamixel"]
+	path = libraries/drivers/dynamixel
+	url = https://github.com/hierophect/Circuitpython_Dynamixel.git
```

Comment:
Can you make the URL somethin like Hierophect_CircuitPython_Dynamixel?

Reply:
Sure. I was trying to figure out the best convention for it from the other libraries in the bundle - I'm not making this on behalf of a personal LLC or anything yet, and it seemed like most other people just left out the prefix if they were doing it independently. I could leave a note on the Guide page that it's recommended to always use a prefix too, if you like.

Reply:
probably for the best, so the authorship is in the title - i can also merge without